### PR TITLE
172 additions to the heat pump quality installation workflows

### PR DESCRIPTION
**Pull request by sudhacheran**
1) Addition to the heat pump workflows as per Edward's request
2) Fix for the project name validation in the modal dialog

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -92,11 +92,11 @@ h1 + h2 {
   margin-bottom: 10px;
 }
 
-/* headding */
-h3 {
+/* heading */
```
Comment (charliepnnl) [inline src/App.css:95]: As a general rule. If we comment out code, we should add a comment about why it is commented out and/or why we might want to uncomment it in the future.

In this case, I think we can just remove this code.

File: src/templates/doe_workflow_central_ducted_split_heat_pump.mdx [hunk 2]
```diff
@@ -107,7 +109,7 @@
         <NumberInput label="Dual Fuel Switch Over Temperature (&deg;F)" min={0} path="dual_fuel_switch_over_temperature" /> 
 
         <PhotoInput id="mfg_extended_cooling_performance_table_photo" label="MFG's cooling performance table – Photo" uploadable>
-            Upload a photo of the MFG's extended cooling performance table showing sensible, latent, or SHR.
+            Upload a photo of the MFG's extended cooling performance table showing sensible, latent, or SHR.
```
Comment (charliepnnl) [inline src/templates/doe_workflow_central_ducted_split_heat_pump.mdx:112]: See https://writingexplained.org/english-abbreviations/manufacturer for the correct way to abbreviate manufacturer.  In the label, it should be "Mfr's". We should write out "manufacturer's" in the description.

Can you search through the template for other instances of this?

**Review by charliepnnl — DISMISSED** [review]
Approved with just a couple of minor suggested changes.

File: src/App.css (hunk 1 above)
Comment (sudhacheran) [reply src/App.css]: For this one, have removed the commented code.

File: src/templates/doe_workflow_central_ducted_split_heat_pump.mdx (hunk 2 above)
Comment (sudhacheran) [reply src/templates/doe_workflow_central_ducted_split_heat_pump.mdx]: updated accordingly in label and description

**Review by charliepnnl — APPROVED** [review]

**Pull request merged**